one/create-storage.sh

```diff
@@ -22,271 +22,87 @@ set -e
 # Use the config file specified in $KUBE_CONFIG_FILE, or default to
 # config-default.sh.
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
```

Comment:
Remove KUBE_ROOT.

There is already a `ABCD_ROOT` which you can use instead of `ROOT`